### PR TITLE
Lageso: max. surveillance cycles

### PR DESCRIPTION
**Pull request by sven-pa**
Lageso: defined max. surveillance cycles per device class (para. 1)

### CODE REVIEW

File: templates/qms/post-market-surveillance/sop-post-market-surveillance.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ Surveillance Plan and Report are created and continuously updated.
 Based on the clinical evaluation and technical documentation, a new Post-Market Surveillance Plan is created
 for a product.
 
+For medical devices of class IIa, post-market surveillance cycles may extend to no longer than two years. For medical devices of classes IIb and III, annual surveillance cycles apply. For devices of class IIa and higher, each surveillance cycle is concluded with the compilation of a periodic safety update report (see below). For class I devices, longer surveillance cycles may be defined.
+
+
```
Comment (olieidel) [inline templates/qms/post-market-surveillance/sop-post-market-surveillance.md]: Remove this newline? We usually only have one newline ("paragraph"?) between sections.

**Review by olieidel — CHANGES_REQUESTED** [review]

File: templates/qms/post-market-surveillance/sop-post-market-surveillance.md (hunk 1 above)
Comment (sven-pa) [reply templates/qms/post-market-surveillance/sop-post-market-surveillance.md]: I'd keep it, as I find several paragraphs in one section (?) often easier to read. Did that also in other SOPs, one example here: https://openregulatory.com/sop-human-resources-administration-template-iso-13485/
But not hard opinion! Your call.

Comment (olieidel) [reply templates/qms/post-market-surveillance/sop-post-market-surveillance.md]: That makes sense! But, when converting markdown files to .docx, .pdf, or just previewing them in Formwork, double-newlines are simply ignored and treated like single-newlines.

See this example (the GitHub editor is also based on markdown):

---

Test

Single newline (there's one empty line above this)


Double newline (there are two empty lines above this - yup, it looks the same)

---

So the question becomes whether you want to keep those double newlines in the markdown files, keeping in mind that they will only be seen there and not being displayed anywhere else (in the preview on our website, in the preview in Formwork, in the pdf, and in the Word file). Your call, haha 😄

Comment (sven-pa) [reply templates/qms/post-market-surveillance/sop-post-market-surveillance.md]: Well, doesn't matter in that case right? Just removed it.

**Review by olieidel — APPROVED** [review]

**Pull request merged**